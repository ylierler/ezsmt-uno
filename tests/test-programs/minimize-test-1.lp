&dom{1..3; 5+3*4} = x:- a, not b.
&dom{1+2..4*4} = x.
{a}.
{b}.
:~ not b.[1]
