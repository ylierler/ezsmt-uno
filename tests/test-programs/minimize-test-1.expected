{}
x=4 
 1

b 
x=4 
 0